packages/botonic-core/src/handoff.js

```diff
@@ -1,40 +1,66 @@
 import axios from 'axios'
 
 export async function getOpenQueues(session) {
-  let base_url = session._hubtype_api || 'https://api.hubtype.com'
-  const queues_url = `${base_url}/v1/queues/get_open_queues/`
-  let bot_id = session.bot.id
+  let baseUrl = session._hubtype_api || 'https://api.hubtype.com'
+  const endpointUrl = `${baseUrl}/v1/queues/get_open_queues/`
   let resp = await axios({
     headers: {
       Authorization: `Bearer ${session._access_token}`
     },
     method: 'post',
-    url: queues_url,
-    data: { bot_id }
+    url: endpointUrl,
+    data: { bot_id: session.bot.id }
   })
   return resp.data
 }
 
-export async function humanHandOff(session, queue_name, on_finish) {
-  let params = `create_case:${queue_name}`
-  if (on_finish) {
-    if (on_finish.path) params += `:__PATH_PAYLOAD__${on_finish.path}`
-    else if (on_finish.payload) params += `:${on_finish.payload}`
+export async function humanHandOff({
```

Comment:
This function in the file `botonic-core/index.d.ts` is declared to return a `Promise<void>`, maybe we should change that declaration? Here we are not returning any `Promise`, right?

Reply:
@guinii async functions always return a promise